web/viewer.js

```diff
@@ -928,10 +928,14 @@ var PDFViewerApplication = {
 
           storedHash = 'page=' + pageNum + '&zoom=' + zoom + ',' +
                        left + ',' + top;
+
+          sidebarView = store.get('sidebarView', SidebarView.NONE);
         } else if (self.preferenceDefaultZoomValue) {
           storedHash = 'page=1&zoom=' + self.preferenceDefaultZoomValue;
         }
-        self.setInitialView(storedHash, scale);
+        self.setInitialView(storedHash, { scale: scale,
```

Comment:
Let's format this block as follows:

<pre>
        self.setInitialView(storedHash,
          { scale: scale, sidebarView: sidebarView });
</pre>